salt/states/docker_container.py

```diff
@@ -304,6 +306,14 @@ def running(name,
                   - /srv/www:/var/www:ro
                   - /home/myuser/conf/foo.conf:/etc/foo.conf:rw
 
+        .. code-block:: yaml
```

Comment:
Adding this here makes the above documentation confusing, as it tells the user that the below two examples are equivalent, and then has three consecutive examples.

Please add a line above this example saying something like "Below is an example of setting the selinux context:"